CONTRIBUTING.md

```diff
@@ -0,0 +1,77 @@
+# Contributing to NetPanzer
+
+## Bug Reports and Feature Requests
+
+Anyone may [open an
+issue](https://github.com/netpanzer/netpanzer/issues).
+
+## Discussion
+
+If you're not sure if something is a bug, you can search, or post on, the
+[Discussions](https://github.com/netpanzer/netpanzer/issues).
+
+### Discord
+
+You may participate on our [Discord server](https://discord.gg/kCAB2CMswd).
+It's important to remember that any discussions about bugs or desired features
+in a chat room usually scroll away and become forgotten, so it's best to
+create a ticket or discussion on GitHub (mentioned above) if you really want a
+bug fix or new feature.
+
+## Coding Standards
+
+Our goal is to use the [Google C++ Style
+Guide](https://google.github.io/styleguide/cppguide.html).
+
+A `.clang-format` file is located in `src`.
+
+You can format code automatically by using
+[clang-format](https://clang.llvm.org/docs/ClangFormat.html) with the
+following arguments:
+
+    clang-format TBD
+
+## Website
+
+TBD
```

Comment:
there are a few sites :)

netpanzer.io (will serve browser-compiled build)
netpanzer.com (main site)
netpanzer.com.br (main site for Brazilian players)
netpanzer.info (old site, probably years before we get this domain)

trying to get them all pointing to the same server, with different localizations.